src/app/[locale]/projects/add/page.tsx

```diff
@@ -7,211 +7,456 @@
 // SPDX-License-Identifier: EPL-2.0
 // License-Filename: LICENSE
 
-"use client";
+'use client'
 
-import styles from "@/css/AddProjects.module.css"
-import AddKeyValueComponent from "@/components/AddKeyValue"
-import SelectCountryComponent from "@/components/SelectCountry"
-import AddAdditionalRolesComponent from "@/components/AddAdditionalRoles"
-import SearchUsersModalComponent from "@/components/SearchUsersModal"
-import Link from "next/link"
+import styles from '@/css/AddProjects.module.css'
+import AddKeyValueComponent from '@/components/AddKeyValue'
+import SelectCountryComponent from '@/components/SelectCountry'
+import AddAdditionalRolesComponent from '@/components/AddAdditionalRoles'
+import SearchUsersModalComponent from '@/components/SearchUsersModal'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
 
 export default function Summary() {
+    const pathName = usePathname()
 
     return (
         <>
             <SearchUsersModalComponent />
-            <div className="mx-5 mt-1">
-                <div className="row mt-2">
-                    <div className="col col-sm-3">
-                        <div className="list-group">
-                            <Link href="/projects/add/summary">Summary</Link>
-                            <Link href="/projects/add/administration">Administration</Link>
-                            <Link href="/projects/add/linkedProjects">Linked Releases and Projects</Link>
+            <div className='mx-5 mt-1'>
```

Comment:
You can use SideBar Component for tab switcher. Refer to Component Detail page
https://github.com/eclipse-sw360/sw360-frontend/blob/main/src/app/%5Blocale%5D/components/detail/%5Bid%5D/components/DetailOverview.tsx